src/main/scala/com/campudus/tableaux/database/domain/column.scala

```diff
@@ -211,6 +417,8 @@ case class ConcatColumn(override val columnInformation: ConcatColumnInformation,
   }
 
   override def getJson: JsonObject = super.getJson mergeIn Json.obj("concats" -> columns.map(_.getJson))
+
+  override def checkValidValue[B](value: B): Try[Option[JsonArray]] = Failure(new IllegalArgumentException("No value allowed for ConcatColumn"))
```

Comment:
This method only gets called when trying to set a value, right? I'd say then "Cannot directly set value for ConcatColumn"